test/Numerics/DGMethods/compressible_navier_stokes_equations/three_dimensional/ThreeDimensionalCompressibleNavierStokesEquations.jl

```diff
@@ -471,31 +466,83 @@ function numerical_flux_first_order!(
     return nothing
 end
 
-## Boundary Conditions
-
 boundary_conditions(model::CNSE3D) = model.boundary_conditions
 
 """
     boundary_state!(nf, ::CNSE3D, args...)
 applies boundary conditions for the hyperbolic fluxes
-dispatches to a function in OceanBoundaryConditions
+dispatches to a function in CNSEBoundaryConditions
 """
 @inline function boundary_state!(nf, bc, model::CNSE3D, args...)
-    return _ocean_boundary_state!(nf, bc, model, args...)
+    return _cnse_boundary_state!(nf, bc, model, args...)
 end
 
 """
-    ocean_boundary_state!(nf, bc::OceanBC, ::CNSE3D)
+    cnse_boundary_state!(nf, bc::FluidBC, ::CNSE3D)
 splits boundary condition application into velocity
 """
-@inline function ocean_boundary_state!(nf, bc::OceanBC, m::CNSE3D, args...)
-    ocean_boundary_state!(nf, bc.velocity, m, m.turbulence, args...)
-    ocean_boundary_state!(nf, bc.temperature, m, args...)
+@inline function cnse_boundary_state!(nf, bc::FluidBC, m::CNSE3D, args...)
+    cnse_boundary_state!(nf, bc.momentum, m, m.turbulence, args...)
+    cnse_boundary_state!(nf, bc.temperature, m, args...)
 
     return nothing
 end
 
 include("bc_momentum.jl")
-include("bc_tracer.jl")
+include("bc_temperature.jl")
+
+"""
+STUFF FOR ANDRE'S WRAPPERS
+"""
+
+function get_boundary_conditions(
+    model::SpatialModel{BL},
+) where {BL <: AbstractFluid3D}
+    bcs = model.boundary_conditions
+
+    west_east = (check_bc(bcs, :west), check_bc(bcs, :east))
+    south_north = (check_bc(bcs, :south), check_bc(bcs, :north))
+    bottom_top = (check_bc(bcs, :bottom), check_bc(bcs, :top))
+
+    return (west_east..., south_north..., bottom_top...)
+end
+
+function DGModel(
+    model::SpatialModel{BL};
+    initial_conditions = nothing,
+) where {BL <: AbstractFluid3D}
+    params = model.parameters
+    physics = model.physics
+
+    Lˣ, Lʸ, Lᶻ = length(model.grid.domain)
+    bcs = get_boundary_conditions(model)
+    FT = eltype(model.grid.numerical.vgeo)
+
+    if !isnothing(initial_conditions)
+        initial_conditions = InitialValueProblem(params, initial_conditions)
+    end
+
+    balance_law = CNSE3D{FT}(
+        initial_conditions,
+        (Lˣ, Lʸ, Lᶻ),
+        physics.advection,
+        physics.dissipation,
+        physics.coriolis,
+        physics.buoyancy,
+        bcs,
+        ρₒ = params.ρₒ,
+        cₛ = params.cₛ,
```

Comment:
`cₛ` does not render in the browser, nor my editor. Can we change the name?

Reply:
<img width="948" alt="Screen Shot 2021-03-04 at 17 56 46" src="https://user-images.githubusercontent.com/7977055/110041987-095ef200-7d13-11eb-86af-0aace0f938af.png">
it displays fine for me?

Reply:
This is what I see:

<img width="1492" alt="Screen Shot 2021-03-05 at 8 58 29 AM" src="https://user-images.githubusercontent.com/1880641/110147833-fa1e8980-7d90-11eb-9f3d-04a2a95b646b.png">

Also, if I copy-paste this into sublime:

<img width="396" alt="Screen Shot 2021-03-05 at 8 59 42 AM" src="https://user-images.githubusercontent.com/1880641/110148030-2c2feb80-7d91-11eb-9a34-4ee1eabab6c4.png">

Reply:
@blallen ?